pkgs/applications/virtualization/virtualbox/guest-additions-iso/default.nix

```diff
@@ -1,22 +1,11 @@
-{ stdenv, fetchurl, lib, virtualbox}:
+{ fetchurl, lib, virtualbox}:
 
 let
   inherit (virtualbox) version;
 in
-stdenv.mkDerivation rec {
-  pname = "VirtualBox-GuestAdditions-iso";
-  inherit version;
-
-  src = fetchurl {
-    url = "http://download.virtualbox.org/virtualbox/${version}/VBoxGuestAdditions_${version}.iso";
-    sha256 = "0efbcb9bf4722cb19292ae00eba29587432e918d3b1f70905deb70f7cf78e8ce";
-  };
-
-  buildCommand = ''
-    mkdir -p $out
-    cp $src $out/
-  '';
-
+fetchurl {
+  url = "http://download.virtualbox.org/virtualbox/${version}/VBoxGuestAdditions_${version}.iso";
+  sha256 = "0efbcb9bf4722cb19292ae00eba29587432e918d3b1f70905deb70f7cf78e8ce";
```

Comment:
Does the meta description work together with fetchurl?

Reply:
Yes, actually, it explictely [accepts the `meta` argument](https://github.com/NixOS/nixpkgs/blob/5c24cf2f0a12ad855f444c30b2421d044120c66f/pkgs/build-support/fetchurl/default.nix#L100), then [passes it to `mkDerivation`](https://github.com/NixOS/nixpkgs/blob/5c24cf2f0a12ad855f444c30b2421d044120c66f/pkgs/build-support/fetchurl/default.nix#L188).